partner_credit_limit/__manifest__.py

```diff
@@ -16,10 +16,11 @@
         able to confirm SO
     ''',
     'depends': [
-        'sale',
+        'sale_management',
     ],
```

Comment:
any reason for a change ?

Reply:
Hello Sir,

In v11 sale module splits in two parts sale and sale_management.
When we install sale module basic sale order table and view are created. But when we install sale_management it allows users to do all sales operation.